### PR TITLE
Stop voice message on redaction

### PR DESCRIPTION
**Pull request by julioromano**
As per product spec: Voice messages must stop playing when redacted.

### CODE REVIEW

File: features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/factories/event/TimelineItemContentRedactedFactory.kt [hunk 1]
```diff
@@ -18,12 +18,29 @@ package io.element.android.features.messages.impl.timeline.factories.event
...
-    fun create(@Suppress("UNUSED_PARAMETER") content: RedactedContent): TimelineItemEventContent {
+    fun create(
+        @Suppress("UNUSED_PARAMETER") content: RedactedContent,
+        eventId: EventId?
+    ): TimelineItemEventContent {
+        eventId?.value?.let {
+            if (mediaPlayer.state.value.mediaId == it) scope.launch(dispatchers.main) {
+                mediaPlayer.pause()
+            }
+        }
```
Comment (bmarty) [inline features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/factories/event/TimelineItemContentRedactedFactory.kt:43]: Weird to do this in this Factory. Is it possible to move this code somewhere else?

Comment (julioromano) [reply features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/factories/event/TimelineItemContentRedactedFactory.kt:43]: Indeed yes, but I'm looking for suggestions. It didn't seem equally good to have to bring this logic all the way to the view layer instead.

Comment (julioromano) [reply features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/factories/event/TimelineItemContentRedactedFactory.kt:43]: Alternative: https://github.com/vector-im/element-x-android/pull/1826

**Comment by julioromano** [comment]
Superseded by https://github.com/vector-im/element-x-android/pull/1826

**Pull request closed without merging**